src/components/strategies/edit/EditStrategyBudgetContent.tsx

```diff
@@ -60,16 +60,16 @@ export const EditStrategyBudgetContent = ({
   } = useLocation();
 
   const calculatedOrder0Budget = !!order0.budget
-    ? new BigNumber(strategy.order0.balance)?.[
+    ? new Decimal(strategy.order0.balance || 0)?.[
```

Comment:
there seems to be something weird here.

We check for `!!order0.budget` but then reference `strategy.order0.balance` - the OR ` || 0` statement shouldnt be necessary.